### ISSUE TITLE
[Feature] Temporarily Show Dynamic Token border

### ISSUE THREAD

**Issue by ChasarooniZ** [MrPrimate/tokenizer#166]
Just want to preface this with, i'm unsure if it's possible/in scope but it would be a neat little QoL

But, my thought is some sort of option that allows you to have a special mask layer, this mask layer acts as a normal mask (IE the mask for Dynamic Token border is already, but with one extra feature. In the Image preview **show the border** (like without the coloration etc.).

If this were possible it makes designing a token more straight forward as it's very clear what the result is

**Comment by MrPrimate** [comment]
Hi, yes, I have been thinking about this, it's just a bit harder than it might seem.

**Comment by ChasarooniZ** [comment]
yeah, I assumed as much. If not, no pressure relatively easy to work around
